src/app.js

```diff
@@ -23,10 +24,31 @@ app.use(bodyParser.json())
 const PORT = process.env.PORT || 3000
 const HOST = process.env.HOST || 'localhost'
 
+// run cache
+let INITIALIZED
```

Comment:
why capital case? its not a const

Reply:
should we write tests as an acceptance test with feature files?